### PR TITLE
fix: MDP-3305 add metadata for images

### PR DESCRIPTION
**Pull request by sheikhimtiaz-cefalo**

### CODE REVIEW

File: src/main/resources/application-development.yml [hunk 1]
```diff
@@ -39,9 +39,18 @@ app:
   maxLimit: 5
   corsSites: "${corsSites:https://devx*,http://localhost*}"
   distributor:
-    dagen: a2c29f44-70f9-4478-9560-5758165e2ede
-    vartland: 5c6469c9-b957-4b3c-9e6b-33ca5f891a39
-    klartale: 02279461-a515-4fa1-a0c5-1130ee595042
-    dagsavisen: 422cadb7-e043-4cb3-a442-3f9737a76936
+    dagen: 64094424-24a7-4df2-98a6-702e15517009
```
Comment (sultanmahmud0) [inline src/main/resources/application-development.yml:42]: Are these production Ids ? I think in the application-development file we should have the Ids of sandbox, not production.

File: src/main/resources/application.yml [hunk 2]
```diff
@@ -42,10 +44,19 @@ app:
   maxLimit: 5
   corsSites: "${corsSites:https://devx*,http://localhost*}"
   distributor:
-    dagen: a2c29f44-70f9-4478-9560-5758165e2ede
-    vartland: 5c6469c9-b957-4b3c-9e6b-33ca5f891a39
-    klartale: 02279461-a515-4fa1-a0c5-1130ee595042
-    dagsavisen: 422cadb7-e043-4cb3-a442-3f9737a76936
+    dagen: 64094424-24a7-4df2-98a6-702e15517009
```
Comment (sultanmahmud0) [inline src/main/resources/application.yml:47]: Are these production Ids ? Also, I guess we should put the Id for morgenbladet.

**Review by sultanmahmud0 — CHANGES_REQUESTED** [review]

File: src/main/resources/application.yml (hunk 2 above)
Comment (sheikhimtiaz-cefalo) [reply src/main/resources/application.yml:47]: FIxed it.

File: src/main/java/no/mentordigital/arctontbuploader/webclient/arc/ArcReaderClient.java [hunk 3]
```diff
@@ -114,19 +115,24 @@ public boolean checkCopyright(Photo photo) {
       log.info("Skipping " + photo.get_id() + " for copyright issue!");
       return false;
     }
-    for (var affiliation : photo.getCredits().getAffiliation()) {
-      if (affiliation.getName().contains(SCANPIX) || affiliation.getName().contains(NTB)) {
-        log.info("Skipping " + photo.get_id() + " for copyright issue!");
-        return false;
+    if (!Objects.isNull(photo.getCredits().getAffiliation())) {
+      for (var affiliation : photo.getCredits().getAffiliation()) {
+        if (affiliation.getName().contains(SCANPIX) || affiliation.getName().contains(NTB)) {
```
[the remark this answers is not in the document]
Comment (sheikhimtiaz-cefalo) [reply src/main/java/no/mentordigital/arctontbuploader/webclient/arc/ArcReaderClient.java:120]: Fixed it.

File: src/main/java/no/mentordigital/arctontbuploader/webclient/arc/ArcReaderClient.java [hunk 4]
```diff
@@ -114,19 +115,24 @@ public boolean checkCopyright(Photo photo) {
...
-    for (var by : photo.getCredits().getBy()) {
-      if (by.getByline().contains(SCANPIX)
-          || by.getByline().contains(NTB)
-          || by.getName().contains(SCANPIX)
-          || by.getName().contains(SCANPIX)) {
-        log.info("Skipping " + photo.get_id() + " for copyright issue!");
-        return false;
+    if (!Objects.isNull(photo.getCredits().getBy())) {
+      for (var by : photo.getCredits().getBy()) {
+        if ((!Objects.isNull(by.getByline()) && !Objects.isNull(by.getName()))
```
[the remark this answers is not in the document]
Comment (sheikhimtiaz-cefalo) [reply src/main/java/no/mentordigital/arctontbuploader/webclient/arc/ArcReaderClient.java:128]: Fixed it.

File: src/main/java/no/mentordigital/arctontbuploader/webclient/arc/model/ImageInStory.java [hunk 5]
```diff
@@ -47,7 +49,8 @@ public ImageMigrationStatus toImageMigrationStatus() {
     return ImageMigrationStatus.builder()
         .arcId(this.imageId)
         .ntbId(this.ntbId)
-        .isSuccess(true)
+        .isSuccess(false)
```
[the remark this answers is not in the document]
Comment (sheikhimtiaz-cefalo) [reply src/main/java/no/mentordigital/arctontbuploader/webclient/arc/model/ImageInStory.java:52]: We are not exporting anything, we are keeping this information initially and updating it once we have migrated the image. Fixed it.

**Review by sheikhimtiaz-cefalo — COMMENTED** [review]
Fixed it

**Review by sultanmahmud0 — APPROVED** [review]
Looks good

**Pull request merged**